samples/runtime-source/zephyr/sample.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_FLASH_RUNTIME_SOURCES=y
+CONFIG_SINGLE_APPLICATION_SLOT=y
+CONFIG_BOOT_SIGNATURE_TYPE_RSA=y
```

Comment:
needs a sample.yml file so that twister will build this

Reply:
would also need adding here https://github.com/mcu-tools/mcuboot/blob/main/.github/workflows/zephyr_build.yaml#L86